conf/cndp_upf_8worker.jsonc

```diff
@@ -0,0 +1,453 @@
+    // entries can be in any order.
+
+    // (R) Application information
+    //    name        - (O) The name of the application
+    //    description - (O) The description of the application
+    "application": {
+        "name": "cndp-upf",
+        "description": "CNDP OMEC-UPF configuration"
+    },
+
+    // (O) Default values
+    //    bufcnt - (O) UMEM default buffer count in 1K increments
+    //    bufsz  - (O) UMEM buffer size in 1K increments
+    //    rxdesc - (O) Number of RX ring descriptors in 1K increments
+    //    txdesc - (O) Number of TX ring descriptors in 1K increments
+    //    cache  - (O) MBUF Pool cache size in number of entries
+    //    mtype  - (O) Memory type for mmap allocations
+    "defaults": {
+        "bufcnt": 32,
+        "bufsz": 2,
+        "rxdesc": 2,
+        "txdesc": 2,
+        "cache": 256,
+        "mtype": "2MB"
+    },
+
+    // List of all UMEM's to be created
+    // key/val - (R) The 'key' is the name of the umem for later reference.
+    //               The 'val' is the object describing the UMEM buffer.
+    //               Multiple umem regions can be defined.
+    // A UMEM can support multiple lports using the regions array. Each lports can use
+    // one of the regions.
+    //    bufcnt  - (R) The number of buffers in 1K increments in the UMEM space.
+    //    bufsz   - (R) The size in 1K increments of each buffer in the UMEM space.
+    //    mtype   - (O) If missing or empty string or missing means use 4KB or default system pages.
+    //    regions - (O) Array of sizes one per region in 1K increments, total must be <= bufcnt
+    //    rxdesc  - (O) Number of RX descriptors to be allocated in 1K increments,
+    //                  if not present or zero use defaults.rxdesc, normally zero.
+    //    txdesc  - (O) Number of TX descriptors to be allocated in 1K increments,
+    //                  if not present or zero use defaults.txdesc, normally zero.
+    //    description | desc - (O) Description of the umem space.
+        "umems": {
+           "umem0": {
+            "bufcnt": 32,
+            "bufsz": 2,
+            "mtype": "2MB",
+            "regions": [
+                            16,16
+                        ],
+            "rxdesc": 0,
+            "txdesc": 0,
+            "description": "UMEM Description 0"
+            },
+           "umem1": {
+            "bufcnt": 32,
+            "bufsz": 2,
+            "mtype": "2MB",
+            "regions": [
+                            16,16
+                        ],
+            "rxdesc": 0,
+            "txdesc": 0,
+            "description": "UMEM Description 1"
+            },
+            "umem2": {
+            "bufcnt": 32,
+            "bufsz": 2,
+            "mtype": "2MB",
+            "regions": [
+                            16,16
+                        ],
+            "rxdesc": 0,
+            "txdesc": 0,
+            "description": "UMEM Description 1"
```

Comment:
Should not this be `2` instead of `1`? Same comment for all of the following descriptions